sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystConf.scala

```diff
@@ -22,6 +22,8 @@ import org.apache.spark.sql.catalyst.analysis._
 private[spark] trait CatalystConf {
   def caseSensitiveAnalysis: Boolean
 
+  def orderByOrdinal: Boolean
```

Comment:
how about `positionAlias`? It's inspired by hive's config: `hive.groupby.orderby.position.alias`.